README.md

```diff
@@ -26,6 +27,9 @@ Best way to start with ESP-BSP is trying one of the [examples](examples) on your
 |---|---|
 | [display](examples/display) | WROVER-KIT |
 | [display_audio](examples/display_audio) | Kaluga-kit |
+| [display_camera](examples/display_camera) | Kaluga-kit |
```

Comment:
Does this example run on the S3-EYE, or not yet?

Reply:
It can be run on S3-EYE too. But currently the user must modify the example's `idf_component.yml` to define dependency on s3-eye BSP and then change one line to include correct BSP header. (same applies to other examples)

We want to focus on user-friendly switching of boards in our upcoming effort